src/terminal.rs

```diff
@@ -80,6 +80,12 @@ fn get_vte_version() -> Option<(u8, u8)> {
     })
 }
 
+/// Checks if the current terminal is WezTerm
```

Comment:
Please do use punctuation in doc comments, here as well as below.  In particular, end sentences with a period.